develop-docs/sdk/data-model/envelope-items.mdx

```diff
@@ -181,18 +181,101 @@ details.
+
+Example payload:
 ```json
-{"event_id":"9ec79c33ec9942ab8353589fcb2e04dc","email":"john@me.com","name":"John Me","comments":"It broke."}\n
+{
+  "event_id": "9ec79c33ec9942ab8353589fcb2e04dc",
+  "timestamp": "2011-05-02T17:41:36Z",
+  "platform": "javascript",
+  "level": "error",
+  "contexts": {
+    "feedback": {
+      "contact_email": "john@example.com",
+      "name": "John Smith",
+      "message": "I love session replay!",
+      "url": "https://sentry.io/replays/",
+      "associated_event_id": "32fd1995636d446385016e2747623e11",
+      "replay_id":"82840977e85b4ed3bc27f7b5b25cec15"
+    }
+  }
+}
 ```
 
-**Attributes**
+**Payload Attributes**
+
+We only document attributes for the `contexts.feedback` object, which is **required**
+for this item type. For other attributes, see [Event Payloads](/sdk/data-model/event-payloads/).
+
+`message`
+
+: **String, required.** Comments of the user, describing what happened and/or sharing
+feedback. The max length is **4096 characters**.
+
+`contact_email`
+
+: *String, recommended.* The email of the user who submitted the feedback.
+
+`name`
+
+: *String, optional.* The name of the user who submitted the feedback.
+
+`url`
+
+: *String, optional.* The URL of the webpage the user was on when submitting feedback.
+  This may be used to search for or set alerts on feedback.
+
+`associated_event_id`
+
+: *UUID String, optional.* The identifier of an error event in the same project.
+  Use this to explicitly link a related error in the feedback UI.
+
+`replay_id`
+
+: *UUID String, optional.* The identifier of a related Session Replay in the same
+  project. Sentry uses this ID to render a Replay clip in the feedback UI.
+
+**Attaching Screenshots:**
+
+You can associate screenshots with a feedback by sending image data as
+[attachment items](/sdk/data-model/envelope-items/#attachment), with `event_id`
+corresponding to the feedback item. We recommend sending the items in the same
+Envelope.
+
+**Constraints:**
+
+- This Item may occur at most once per Envelope.
+- This Item is mutually exclusive with `"transaction"` items.
+
+**Required Envelope Headers:**
 
 `event_id`
 
-: **UUID String, required.** The identifier of the event or transaction.
+: **UUID String, required.** Corresponds to the `event_id` field of the event
+  payload. **It is not equal to the `associated_event_id`** field in the feedback
+  context. Clients are required to generate an event identifier ahead of time
+  and set it at least in the Envelope headers. If the identifier mismatches
+  between the Envelope and payload, the Envelope header takes precedence.
+
+### User Report
 
+Item type `"user_report"`. This is an older way of submitting user feedback we
+are in the process of deprecating. It works by associating user information and
+comments with an event. If both the item and its associated event are accepted,
+we convert it to a user feedback.
```

Comment:
```suggestion
Item type `"user_report"`. This is an older way of submitting user feedback we
have now deprecated it. It will continue to work, so older SDKs won't break. But new implementation should follow the User Feedback `feedback` type described above.

It works by associating user information and
comments with an event. If both the item and its associated event are accepted,
we convert it to a user feedback.
```